processmanager/helpers.go

```diff
@@ -79,3 +90,36 @@ type FileIDMapper interface {
 	// Set adds a mapping from the 64-bit file ID to the 128-bit file ID.
 	Set(pre host.FileID, post libpf.FileID)
 }
+
+func parseContainerID(cgroupFile io.Reader) string {
```

Comment:
Since the tests only contain cgroupV2 strings, should we document this function to support cgroupV2 only? Otherwise, we should add cgroupV1 test cases.
IMO, we don't need to support for v1.